src/runtime/markdown-parser/content.ts

```diff
@@ -12,8 +12,7 @@ import { nodeTextContent } from './utils/node'
 const usePlugins = (plugins: Record<string, false | MarkdownPlugin>, stream: Processor) => {
   for (const plugin of Object.values(plugins)) {
     if (plugin) {
-      const { instance, ...options } = plugin
-      stream.use(instance, options)
+      stream.use(plugin.instance, plugin.options)
```

Comment:
This will introduce a breaking change for standalone usage.

I think it is better to create a special case for `options` field and add a warning. We may remove old syntax in next releases.

```suggestion
      const { instance, options, ...deprecatedOptions } = plugin
      if (Object.keys(deprecatedOptions).length) {
        console.warn('[Markdown] Deprecated syntax. Please use `options` key in order to pass option to remark/rehype plugins.')
      }
      stream.use(instance, options ? options : deprecatedOptions)
```

Reply:
@farnabaz sorry, I accepted your suggestion without renaming the commit 🙈